packages/browser-wallet/src/popup/store/utils.ts

```diff
@@ -21,6 +21,7 @@ import {
     sessionRecoveryStatus,
     sessionIdpTab,
 } from '@shared/storage/access';
+import { pendingTransactions } from '@shared/storage/transactions';
```

Comment:
I think this should be in `@shared/storage/access` along with the rest to keep it all together.